community/CloudflareR2.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: page
+title: Hosting BlueMap on Cloudflare R2
+parent: Community Guides
+nav_order: 15
+---
+
+# How to host BlueMap on Cloudflare R2
+{: .no_toc }
+
+1. TOC
+{:toc}
+
+## What is Cloudflare R2?
+
+Cloudflare R2 is a service that allows you to host files on Cloudflare's network. With a couple of tweaks, you can host static websites such as BlueMap on Cloudflare R2. Please bear in mind that Cloudflare R2 costs money.
+
+## What do you need for this?
+
+- Cloudflare account
+- Domain you own & is connected with Cloudflare
+- Credit card/prepaid card you have
+- a completely rendered BlueMap map (you can use the CLI or the docker container for that) — this is intended as a one-time upload for a map where further adjustments are unnecessary
+- rclone on the system you've rendered the map on
+
+
+## Adding a domain to Cloudflare
+
+(You can skip this part and go to Setting Zero Trust Dashboard part)
+
+Go to Websites, click on “Add a site” then write your domain name.
+
+1. “Select your plan”
+    - You can choose Free plan at the bottom.
+
+2. “Review DNS records”
+    - You can skip it because we don't need to set a DNS for this method.
+
+3. “Change your nameservers”
+    - Go to the site you bought your domain and change nameservers to Cloudflare's nameservers.
+
+
+## Creating a bucket & uploading files with rclone
+
+1. Go to R2, click on “Create bucket” then write your bucket name.
+
+2. In your new bucket, go to settings.
+
+3. Scroll to Public Access / Custom Domains and click on “Connect Domain”. Enter the domain you want to use for BlueMap and confirm the changes.
+
+4. (optional) Scroll down and edit the “Default Multipart Abort Rule” by clicking on the three dots and then “Edit”. Set the “Multipart Abort Rule” to 1 day.
```

Comment:
What does this do? And why is it optional?

Reply:
Changed✔ Added a short explanation.